manifests/jenkins-s2i.yaml

```diff
@@ -14,7 +14,7 @@ parameters:
     value: main
   - description: Source imagestream
     name: JENKINS_S2I_SRC_IMAGESTREAM_NAME
-    value: jenkins:2
+    value: jenkins:scheduled-upgrade-redeploy
```

Comment:
does the below asciiart need to be updated?

Reply:
ah, thanks for catching that. Yes, I'll update that.

Reply:
Updated!